person.rb

```diff
@@ -1,6 +1,7 @@
 require_relative 'nameable'
 require_relative 'capitalize_decorator'
 require_relative 'trimmer_decorator'
+require_relative 'rentable'
 
 class Person < Nameable
   attr_accessor :name, :age
```

Comment:
- [ ] Kindly define the `@rentals` list as `attr_accessor` as you can access it inside the initialize method Rental class.

Reply:
done